experimental/rust/image/Dockerfile-stack

```diff
@@ -20,9 +20,12 @@ ENV APPSODY_DEBUG="cargo build && lldb-server platform --listen '*:1234' --min-g
 ENV APPSODY_DEBUG_ON_CHANGE="$APPSODY_DEBUG"
 ENV APPSODY_DEBUG_KILL=true
 
-COPY ./LICENSE /licenses
-COPY ./project /project
-COPY ./config /config
+RUN useradd rust
+USER rust
+
+COPY --chown=rust:rust ./LICENSE /licenses
+COPY --chown=rust:rust ./project /project
+COPY --chown=rust:rust ./config /config
```

Comment:
copy --chown is not supported in older docker versions. We should add the requirement for that in stack.yaml.